packages/circuits/circuits/epochKey.circom

```diff
@@ -29,6 +29,8 @@ template EpochKey(STATE_TREE_DEPTH, EPOCH_KEY_NONCE_PER_EPOCH, FIELD_COUNT) {
 
     // Some arbitrary data to endorse
     signal input sig_data;
+    // dummy square to ensure constraint
+    signal sig_data_square <== sig_data * sig_data;
```

Comment:
It can be done in `epochKeyLite` template?
Since it is an input signal 

https://github.com/Unirep/Unirep/blob/a3d0cb31b34b754a9acf31783686cbfb4277f4f5/packages/circuits/circuits/epochKey.circom#L67